api/v2alpha1/go/cachekey/cache_key.pb.go

```diff
@@ -14,7 +14,7 @@
 
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
-// 	protoc-gen-go v1.26.0
+// 	protoc-gen-go v1.33.0
```

Comment:
can you generate using proto-gen-go v1.26.0?

Reply:
slash that, we'll want to actually update this to match what's in the api-generator image, but let's do this as a separate pr